phost/src/main.rs

```diff
@@ -245,7 +303,11 @@ async fn bridge(args: Args) -> Result<(), Error> {
         let mut blocks = Vec::<phala_node_runtime::SignedBlock>::new();
         for h in info.blocknum ..= block_tip.block.header.number {
             let block = get_block_at(&client, Some(h)).await?;
-            blocks.push(block);
+            blocks.push(block.clone());
+
+			if h > 0 {
+				let _ = sync_events(&client, &block, h).await;
+			}
         }
```

Comment:
The logic here is incorrect. We shouldn't sync the event before syncing the block. And another problem is that, the blocks are packed and synced at once, but the events are synced one by one. I think it makes more sense if we sync the events at the same time syncing the blocks.